README.md

```diff
@@ -48,12 +48,13 @@ Unstable testing versions of the lates builds of the configurator for most platf
 
 **Be aware that these versions are intended for testing / feedback only, and may be buggy or broken, and can cause flight controller settings to be corrupted. Caution is advised when using these versions.**
 
-## Native app build via NW.js
+## Native app build via NW.js (windows/linux/macos) or Cordova (android)
 
 ### Development
 
 1. Install node.js (version 10 required)
 2. Install yarn: `npm install yarn -g`
+3. (For Android platform only) Install Java JDK 8, Gradle and Android Studio (Android SDK at least level 16)
```

Comment:
These requirements seem to be missing a key part: Cordova. Or better yet, add cordova under `devDependencies` in `package.json`.

Apart from that I am slowly getting there, the build is currently failing with this error:

```
[03:51:18] Starting 'cordova_build'...
Checking Java JDK and Android SDK versions
ANDROID_SDK_ROOT=undefined (recommended setting)
ANDROID_HOME=/opt/android-sdk/ (DEPRECATED)
Reading build config file: /home/mikeller/git/betaflight-configurator/cordova/build.json
Reading the keystore from: /home/mikeller/git/betaflight-configurator/cordova/release.keystore
:wrapper

BUILD SUCCESSFUL in 0s
1 actionable task: 1 executed
Subproject Path: CordovaLib
Subproject Path: app
> Task :app:preBuild UP-TO-DATE
> Task :CordovaLib:preBuild UP-TO-DATE
> Task :CordovaLib:preReleaseBuild UP-TO-DATE
> Task :CordovaLib:checkReleaseManifest
> Task :CordovaLib:processReleaseManifest
> Task :app:preReleaseBuild FAILED
3 actionable tasks: 3 executed


FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':app:releaseRuntimeClasspath'.
> Could not resolve com.android.support:support-v4:+.
  Required by:
      project :app
   > Failed to list versions for com.android.support:support-v4.
      > Unable to load Maven meta-data from https://jcenter.bintray.com/com/android/support/support-v4/maven-metadata.xml.
         > Could not get resource 'https://jcenter.bintray.com/com/android/support/support-v4/maven-metadata.xml'.
            > Could not GET 'https://jcenter.bintray.com/com/android/support/support-v4/maven-metadata.xml'. Received status code 504 from server: Gateway Time-out

Reply:
I don't know why it doesn't work.
Cordova uses a different `package.json` located in the cordova folder. And cordova is under `dependencies` in this package. So we don't need to add cordova in the `package.json` used by gulp and the rest of the configurator.

Reply:
> I don't know why it doesn't work.

Still working on that - seems to be more of a systemic problem, just curious that it's working for you but not for me. What platform are you building on?

> Cordova uses a different package.json located in the cordova folder. And cordova is under dependencies in this package. So we don't need to add cordova in the package.json used by gulp and the rest of the configurator.

There's a chicken and egg problem in there - if cordova is not already available to the top-level build process, then the cordova process can not be started. So either cordova needs to be pre-installed on the system as a prerequisite, or it needs to be added to the top level `package.json` `devDependencies` list so that it is installed and made available when the cordova part of the build is started.

Reply:
I build on Windows. I try to build again on Linux to see if I have the same issue as you.

Reply:
Never mind, when I tried again today it all worked - looks like there was a transient network problem when I tried last time.
Got it running on android now, and I am impressed! Still quite a few UI snags and that will need to be addressed (mostly because the current styling is entirley not designed for this), but all in all it works, which is great.
I still think that cordova should be added as a top level `devDependency` to avoid requiring users to have cordova installed on their machines.

Reply:
And to clarify, I did not mean that the UI snags need to be addressed before this can be merged - just before the android version becomes part of the official release - I imagine doing this bit by bit and tab by tab is probably the only viable approach.

Reply:
I think, when all features will work on Android, it could be interesting to make all the UI responsive design. And I added cordova as `devDependency`.

Reply:
Yeah, I think responsive will be the way to go - but it will be a lot of work to get there, looking at the 'organically grown' nature of the current UI.